cgroup1/cpuacct.go

```diff
@@ -34,11 +35,13 @@ var clockTicks = getClockTicks()
 func NewCpuacct(root string) *cpuacctController {
 	return &cpuacctController{
 		root: filepath.Join(root, string(Cpuacct)),
+		buf:  bytes.NewBuffer(make([]byte, 0, 32)),
```

Comment:
I like the idea! 

I'm not sure how many users are using single subsystem. If we agree with this change, we should add comment about `Stat` function with `it is not thread-safety` as warning.

cc @AkihiroSuda

Reply:
It'd be any of the methods that call `readUint` not just Stat, but yea either a warning or some way to avoid this https://github.com/containerd/cgroups/pull/275#discussion_r1148809008

Reply:
I wanted to get try this.
See: https://github.com/containerd/cgroups/pull/277